flink-core/src/main/java/org/apache/flink/configuration/RestOptions.java

```diff
@@ -37,6 +37,17 @@
 			.withDeprecatedKeys(WebOptions.ADDRESS.key(), ConfigConstants.DEFAULT_JOB_MANAGER_WEB_FRONTEND_ADDRESS.key())
 			.withDescription("The address that the server binds itself.");
 
+	/**
+	 * The port range that the server could bind itself to.
+	 */
+	public static final ConfigOption<String> BIND_PORT =
+		key("rest.bind-port")
```

Comment:
this _must_ use `rest-port` as a deprecated key

Reply:
You are right.